client/jetpack-cloud/sections/overview/primary/next-steps/index.tsx

```diff
@@ -123,7 +123,11 @@ export default function NextSteps( { onDismiss = () => {} } ) {
 					</button>
 				</p>
 			) }
-			<Checklist tasks={ tasks } />
+			<Checklist>
+				{ tasks.map( ( task ) => (
+					<ChecklistItem task={ task } key={ task.id } onClick={ task.actionDispatch } />
+				) ) }
+			</Checklist>
```

Comment:
See my comments in `ChecklistItem` around making `task.actionDispatch` be the fallback for `onClick`.

Reply:
Answered on the previous comment